generated/ruby/test/errorreporting.rb

```diff
@@ -0,0 +1,160 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Integration tests for vkit generated client library for
+# Stackdriver ErrorReporting APIs.
+#
+require "google/cloud/error_reporting/v1beta1"
+
+PROJECT_NAME = "gapic-test".freeze
```

Comment:
Is it okay to embed this project name as a constant? Can we get the name from the environment variable (i.e. `ENV['PROJECT_ID']`?)